src/corehost/build.sh

```diff
@@ -184,6 +184,13 @@ else
     exit 1
 fi
 
+if [ -z $CONFIGURATION ]; then
+    export CONFIGURATION=Debug
+fi
+echo "CONFIGURATION=$CONFIGURATION"
+
+__cmake_defines="-DCMAKE_BUILD_TYPE=${CONFIGURATION} ${__cmake_defines}"
```

Comment:
Nope :) I think it's enough to fix issue #1386. I will close this PR and issue #1386 soon after look into CI result. Thank you for the fix.